src/sentry/interfaces/stacktrace.py

```diff
@@ -428,10 +427,30 @@ def get_hash(self, platform=None, is_processed_data=True):
                 output.append(remove_function_outliers(self.function))
         elif self.lineno is not None:
             output.append(self.lineno)
-            if not is_processed_data and self.colno is not None:
-                output.append(self.colno)
         return output
 
+    def _get_unprocessed_hash(self, platform=None):
+        platform = self.platform or platform
+        if platform == 'javascript':
+            # Safari throws [native code] frames in for calls like ``forEach``
+            # whereas Chrome ignores these. Let's remove it from the hashing algo
+            # so that they're more likely to group together
+            if self.filename == '[native code]':
+                return []
+
+            attrs = [self.filename, self.function, self.colno, self.lineno]
+            if all(attrs):
+                return attrs
+            else:
+                return []
```

Comment:
I don't think we should drop these. We should just consider these unhashable instead.

Reply:
would returning an empty hash set not signal unhashable? if not it probably should

Reply:
@dcramer returning an empty hash would probably result in a lower priority interface (like the message) being used instead (rather than our being able to handle this and continue with processing so it doesn't wind up matching an incorrect group)